js/main.js

```diff
@@ -146,3 +193,15 @@ app.on('activate', function () {
         windowMgr.showMainWindow();
     }
 });
+
+app.setAsDefaultProtocolClient('symphony');
+
+app.on('open-url', function (event, url) {
+    if (!isAppAlreadyOpen) {
```

Comment:
that is what I had thought as well until I started testing and found out that electron emits this event before the 'ready' event when the app is not open. So, we'll need to handle both the cases here.